benchmark.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```

Comment:
I would do `#!/usr/bin/env python3` here just to be safe.

Reply:
fine with me.

Reply:
@slavong Could you also please do this so it can be merged.